### ISSUE TITLE
Parser breaks after html tag

### ISSUE THREAD

**Issue by grieverrr** [phug-php/phug#34]
Hello,

I encountered an issue with the following code:
```pug
div.grid.pg-header__grid
  div.pg-header__top
  <span>test</span>
    div.pg-header__city

```

I expected to get:
```html
<div class="grid pg-header__grid">
  <div class="pg-header__top"><span>test</span>
    <div class="pg-header__city"></div>
  </div>
</div>

```

But I actually get:
```html
<div class="grid pg-header__grid">
  <div class="pg-header__top"><span>test</span>
    div.pg-header__city
</div>
</div>

```

Thanks!

**Comment by kylekatarnls** [comment]
It's the expected behavior:
```pug
<p>
  Indented text is raw text
</p>
```
However the following is expected to get what you want:
```pug
div.grid.pg-header__grid
  div.pg-header__top
    <span>test</span>
    div.pg-header__city
```
Note that however <span> Must be indented, else it's not inside pg-header__top.

I will inspect.

**Comment by kylekatarnls** [comment]
Right now, I have only this as work-around:
```pug
div.grid.pg-header__grid
  div.pg-header__top
    | <span>test</span>
    div.pg-header__city
```

**Comment by grieverrr** [comment]
It is not really problem, but the thing is confusing is that the pugjs have different behavior in this case

**Comment by kylekatarnls** [comment]
I finally aligned this behavior on pugjs. You can test the next release of the lexer by running `composer require phug/lexer=dev-master` (need a dev minimum stability), or you can cheat your **composer.json** to not change the minimum stability:
```json
    "require": {
        "phug/lexer": "dev-master as 0.5.999",
    },
```

When I will tag it as official release and test it via other project you'll be able to remove the lexer from your dependencies.

**Comment by kylekatarnls** [comment]
Fixed in lexer 0.5.27
